stage1/usr_from_coreos/usr_from_coreos.mk

```diff
@@ -68,6 +68,7 @@ $(UFC_ACI_ROOTFS_STAMP): $(UFC_MKBASE_STAMP) $(UFC_FILELIST)
 	ln -sf 'usr/lib64' "$(ACIROOTFSDIR)/lib64"; \
 	ln -sf 'usr/lib' "$(ACIROOTFSDIR)/lib"; \
 	ln -sf 'usr/bin' "$(ACIROOTFSDIR)/bin"; \
+	mkdir "$(ACIROOTFSDIR)/proc"; \
```

Comment:
Directories and symlinks are stupid things to handle in make.

Let's say that it is fine for now, but it might require doing a clean rebuild now. But please add `$(ACIROOTFSDIR)/proc` to CLEAN_DIRS too.